src/scss/modifiers/_inverse.scss

```diff
@@ -29,5 +28,75 @@
 				border-color: _oFormsGet('toggle-inverse-selected');
 			}
 		}
+
+		.o-forms-input--invalid {
+			*:invalid:not(:disabled) {
+				color: _oFormsGet('invalid-base-color-inverse');
+				border-color: _oFormsGet('invalid-base-border-inverse');
+
+				+ .o-forms-input__label:before,
+				+ .o-forms-input__label:after {
+					border-color: _oFormsGet('invalid-base-border-inverse');
+				}
+			}
+
+			.o-forms-input__error {
+				color: _oFormsGet('invalid-base-inverse');
+			}
+		}
+
+		.o-forms-input--radio-box {
+			border-color: _oFormsGet('default-border');
+
+			.o-forms-input--radio-box__container,
+			label:not(:first-of-type) {
+				border-color: _oFormsGet('default-border');
+				background-color: _oFormsGet('default-background');
+			}
+
+			input[type=radio] { // stylelint-disable-line selector-no-qualifying-type
+				& + .o-forms-input__label { // stylelint-disable-line selector-no-qualifying-type
+					color: _oFormsGet('controls-base');
+				}
+
+				&:not(:checked):not(:disabled):hover + .o-forms-input__label { // stylelint-disable-line selector-no-qualifying-type
+					background-color: rgba(_oFormsGet('controls-base'), 0.15);
+				}
+
+				&:focus + .o-forms-input__label { // stylelint-disable-line selector-no-qualifying-type
+					border-color: oColorsByUsecase('focus', 'outline', $fallback: null);
```

Comment:
Do you know in what case the fallback is hit? Whitelabel brand? Is there be an acceptable focus state without?

Reply:
Good question! It's actually code which isn't needed in this PR so I've now removed it, it's a line in the checkbox styles -- https://github.com/Financial-Times/o-forms/blob/6cc1d7f30579f251ed596c4292df05eed97d3435/src/scss/_checkbox.scss#L15